kuma/users/views.py

```diff
@@ -516,7 +502,10 @@ def get_form(self, form_class):
                     label = _('%(email)s Unverified')
                 next_email = email_address['email']
                 choices.append((next_email, label % {'email': next_email}))
-            choices.append((form.other_email_value, _('Other:')))
+            if extra_email_addresses:
+                choices.append((form.other_email_value, _('Other:')))
+            else:
+                choices.append((form.other_email_value, _('Email:')))
```

Comment:
This is a feature request, out of scope for this PR. Please open a bug:

https://bugzilla.mozilla.org/enter_bug.cgi?product=Mozilla%20Developer%20Network&component=Profiles#h=detail%7Cchange